ou_dedetai/system.py

```diff
@@ -259,7 +260,7 @@ def reboot():
     sys.exit(0)
 
 
-def t(command):
+def t(command): # FIXME: unused, have_dep does the same thing
```

Comment:
I've noticed this before but didn't give it much thought. Good to replace one with another.